### DISCUSSION TITLE
Inquiry about Column Chart's sacle

### DISCUSSION THREAD

**Discussion by siyingch86** [patrykandpatrick/vico#645] [Questions] [answered]
Hi, wanna know is there any way to show each bar with different scale? In the column chart, when one bar is extremely higher, some bars will be visibly minimum. Is there any way to apply different scale for each bar, or setting a minimum value for a bar so that when it is below that value, show it higher? 

<img width="29" alt="Screenshot 2024-04-04 at 9 13 23 PM" src="https://github.com/patrykandpatrick/vico/assets/160801134/9f4600ef-dddb-482f-8ed1-e6c95b8d7bf5">

**Comment by Gowsky** [comment, marked as answer]
Hello, does it relate to your other [comment](https://github.com/patrykandpatrick/vico/discussions/633#discussioncomment-9022691)? If so, did [this](https://github.com/patrykandpatrick/vico/discussions/633#discussioncomment-9030301) answer your question?

**Reply by siyingch86** [reply]
yes it should answer

**Reply by patrickmichalik** [reply]
Thanks for confirming!
